todocore/src/main/java/com/knighttodo/todocore/service/DayService.java

```diff
@@ -53,4 +56,9 @@ public void deleteById(UUID dayId) {
         dayRepository.softDeleteAllDayTodosByDayId(dayId);
         dayRepository.softDeleteById(dayId);
     }
+
+    @Transactional
```

Comment:
consider using read-only parameter for get requests

Reply:
Done!

Reply:
If you have @Transactional(readOnly=true) on the class level you don't need to put it on the method level because you will use Transactional without readOnly=true this way

Reply:
Ok, but it is not right to use @transactional(readOnly=true) for all class because we have methods which modifying  data  in DB. And must we set @transactional(readOnly=true) only for FIND methods which only read data from DB?